src/v/kafka/server/handlers/fetch.h

```diff
@@ -161,22 +161,17 @@ struct fetch_config {
 };
 
 struct ntp_fetch_config {
```

Comment:
this change makses `ntp_fetch_config` not movable

Reply:
how so? `materialized_ntp` is movable